Implement Feedback block

izmalk opened the pull request:
Adding the Feedback block just as in the Furo-based theme, but adjusting it's location and using the Vanilla Framework icons.

<img width="1930" height="724" alt="image" src="https://github.com/user-attachments/assets/fe748d2d-c9c5-488f-8df2-edb1106eb3cd" />

Also, adjusted the top margin of the content and side menus to be more aligned.

izmalk committed:
Initial version

izmalk committed:
Initial version

izmalk committed:
First decently looking version

izmalk committed:
First decently looking version

izmalk committed:
Content and side menus top margin  + refactoring

Add a top margin to all content, including the side menus.
Refactor feedback CSS styles for size and readability.

izmalk committed:
Content and side menus top margin  + refactoring

Add a top margin to all content, including the side menus.
Refactor feedback CSS styles for size and readability.

izmalk committed:
Implement logic and refactor

Refactored for new grid layout changes.
Implemented the correct edit/view buttons logic/URLs.

izmalk committed:
Implement logic and refactor

Refactored for new grid layout changes.
Implemented the correct edit/view buttons logic/URLs.

izmalk committed:
Fixing a local TOC design issue

A small issue appears on certain window size - feedback block wraps the last icon to the next line. This fixes the problem.

izmalk committed:
Fixing a local TOC design issue

A small issue appears on certain window size - feedback block wraps the last icon to the next line. This fixes the problem.

izmalk committed:
Merge branch 'main' into give-feedback

nbellol commented on ulwazi/theme/ulwazi/sections/feedback.html:
```diff
@@ -0,0 +1,50 @@
+<div class="content-icon-container">
+  <div class="github-issue-link-container">
+    <a class="muted-link github-issue-link"
+       href="https://github.com/canonical/kafka-operator/issues/new?title=docs%3A+TYPE+YOUR+QUESTION+HERE&amp;body=*Please describe the question or issue you're facing with &quot;Charmed Apache Kafka documentation&quot;.*%0A%0A%0A%0A%0A---%0A*Reported+from%3A+https://documentation.ubuntu.com/charmed-kafka/3/*"
```
All items take to Kafka
<img width="1387" height="666" alt="image" src="https://github.com/user-attachments/assets/bd6ddfc1-edaf-4eb0-9a7f-e3b072b1c7ab" />

izmalk replied:
That was the case for the previous version, last week. Can you please try again with resetting the environment?
Just `make clean` should do. I would also do the `make vanilla-main` after that, just in case.
You can see that the PR preview works correctly.

izmalk replied:
I've localised the problem and fixed in https://github.com/canonical/ulwazi/pull/39/commits/e8bf9ccd76011566076ca397ca762266e31c17c9
The problem was only with the Give feedback button. The edit and view source buttons were working fine.

nbellol reviewed (changes requested):
Everything looks good and works. Except giving feedback always navigate to kafka

izmalk committed:
Update contribution and usage guides (#45)

Added content for the contribution guide.
Added a simple usage guide.

izmalk committed:
Fix the Give feedback button link

izmalk committed:
Fix the Give feedback button link

a-velasco committed:
Add PDF smoke test (#44)

Used explicit sphinx target for admonition reference.
Added a minimal PDF test.

izmalk committed:
Fix issue text template

Fixed project name in the issue template

izmalk committed:
Fix issue text template

Fixed project name in the issue template

nbellol reviewed (approved):
LGTM

geoffreynyaga committed:
Implement Vanilla styles for tabs (#35)

* theme: implement vanilla styles for tabs
* [fix] Refactor sphinx-tabs

geoffreynyaga committed:
Vanilla Framework lists styles implementation for rST and Markdown (#24)

* lists implementation with definition list support

izmalk committed:
Rebuild CSS

izmalk committed:
Merge branch 'give-feedback' of github.com:canonical/ulwazi into give-feedback
